includes/ppc-frontend.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Pre-Publish Checklist php for Displaying the contents in the general tab in the plugin settings
+ * PHP version 7
+ *
+ * @category PHP
+ * @package  Pre-Publish Checklist.
+ * @author   Display Name <username@ShubhamW.com>
+ * @license  http://brainstormforce.com
+ * @link     http://brainstormforce.com
+ */
+
+// Displaying the contents in the general tab in the plugin settings.
+$ppc_radio_button        = get_option( 'ppc_radio_button_option_data' );
+$ppc_radio_button        = ( ! empty( $ppc_radio_button ) ? $ppc_radio_button : 3 );
+$ppc_checklist_item_data = get_option( 'ppc_checklist_data' );
```

Comment:
This variable does not seem to be used anywhere, can you verify and remove this?